devtools-ci-index.yaml

```diff
@@ -1798,6 +1806,12 @@
                 - master
     <<: *job_template_build_defaults
 
+- job-template:
+    name: '{ci_project}-launcher-backend'
+    wrappers:
+        - fabric8_launcher_credentials_wrapper
+    <<: *job_template_build_defaults
+
```

Comment:
when i run it with job-builder here on my machine, this does not seem to replace the right credential in.